kernel/sched/rr.c

```diff
@@ -87,10 +87,13 @@ int sched_rr_elect(int switch_type)
 }
 
 // clang-format off
-const struct sched sched_rr = {
+static struct sched sched_rr = {
+    .cls = SCHED_CLASS_RR,
```

Comment:
It would be `.class` or `.policy`.

Reply:
These confuse me a lot. In my opinion and consideration of this change, I abstract `struct sched` as a sched class and add a class member, `policy`. Moreover, I'll replace macro `SCHED_CLASS_XXX` to `SCHED_POLICY_XXX`.